.eslintrc

```diff
@@ -1,5 +1,8 @@
 {
     "extends": [
         "./eslint/eslintrc"
-    ]
+    ],
+    "rules": {
+        "flowtype/require-exact-type": [2, "always"]
+    }
```

Comment:
I think eventually we'll want to back port some (all?) of these new rules to khan-linter.  There's quite a lot of process in doing so so unfortunately I don't see that happening any time soon. 😞